BondageClub/Scripts/VariableHeight.js

```diff
@@ -72,8 +72,8 @@ function VariableHeightCreateData(asset,
 			CommonChatTags.TARGET_CHAR,
 			CommonChatTags.ASSET_NAME
 		],
-		getHeight: window[GetHeightFunction] || VariableHeightDefaultGet,
-		setHeight: window[SetHeightFunction] || VariableHeightDefaultSet,
+		getHeight: GetHeightFunction,
+		setHeight: SetHeightFunction,
```

Comment:
One further thought on this - I feel like it would probably be neater to have the `VariableHeightGetOverrideHeight`/`VariableHeightSetOverrideHeight` functions be the default values for this so that people don't need to explicitly define it, as I imagine the majority of cases are going to use these default implementations.
```suggestion
		getHeight: GetHeightFunction || VariableHeightGetOverrideHeight,
		setHeight: SetHeightFunction || VariableHeightSetOverrideHeight,
```

Reply:
I agree and had planned on doing so as soon as we had even one different function in use, so that these could serve as a demonstration of how the config should/can be done. Realistically that mightn't happen any time soon and would probably require assistance anyway, so yeah may well make that change now.